tools/ports/libpng.py

```diff
@@ -31,7 +31,7 @@ def get_lib_name(settings):
 
 def get(ports, settings, shared):
   # This is an emscripten-hosted mirror of the libpng repo from Sourceforge.
-  ports.fetch_project('libpng', f'https://storage.googleapis.com/webassembly/emscripten-ports/libpng-{TAG}.tar.gz', sha512hash=HASH)
+  ports.fetch_project('libpng', f'https://download.sourceforge.net/libpng/libpng-{TAG}.tar.gz', sha512hash=HASH)
```

Comment:
We had trouble in the past with download.sourceforge.net being unreliable (we need to to be up constantly to test this in CI).

Reply:
Thanks; that's probably the source of the CI issues.
Can you please upload a copy of the libpng-1.6.39.tar.gz to the Google Storage repo? I'll revert that change.

Reply:
The CI issue actually looks different, I think you need to update the pngtest in the tests directory.

Reply:
I swapped that back and fixed the pngtest test.